examples/manpage.rs

```diff
@@ -3,9 +3,8 @@ use libc::{self, c_void};
 use nix::poll::{poll, PollFd, PollFlags};
 use nix::sys::mman::{mmap, MapFlags, ProtFlags};
 use nix::unistd::{sysconf, SysconfVar};
-use std::env;
 use std::os::unix::io::AsRawFd;
-use std::ptr;
+use std::{convert::TryInto, env};
```

Comment:
Note that if the crate was edition 2021 we wouldn't need to explicitly import `TryInto`: https://doc.rust-lang.org/edition-guide/rust-2021/prelude.html

Reply:
Thanks. If you'd like to follow up with a PR to upgrade to edition 2021 that would be fine, but its not a big worry either way.